### PR TITLE
first attempt

### PR DESCRIPTION
**Pull request by jcommelin**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by jcommelin — COMMENTED** [review]
maintainer merge

**Review by jcommelin — COMMENTED** [review]
with extra text

maintainer merge

**Comment by jcommelin** [comment]
as a regular comment

maintainer merge

**Comment by jcommelin** [comment]
maintainer merge

on the first line

File: .github/workflows/maintainer_merge_review.yml [hunk 1]
```diff
@@ -20,9 +20,9 @@ jobs:
           type: 'stream'
           topic: 'maintainer merge'
           content: |
-            ${{ format('{0} requested a maintainer merge on PR #{1}:', github.event.review.user.login, github.event.review.pull_request.id) }}
+            ${{ format('{0} requested a maintainer merge on PR #{1}:', github.event.review.user.login, github.event.pull_request.number) }}
 
-            > ${{ github.event.review.issue.title }}
+            > ${{ github.event.pull_request.title }}
```
Comment (bryangingechen) [inline .github/workflows/maintainer_merge_review.yml:25]: maintainer merge

**Pull request merged**